runtime/src/iree/modules/vmvx/module.c

```diff
@@ -44,6 +44,11 @@ typedef struct iree_vmvx_module_t {
 typedef struct iree_vmvx_module_state_t {
   iree_allocator_t host_allocator;
 
+  // Logical processor identifier used to index into processor info fields.
+  // Depending on the implementation this may be an ordinal, a bitfield, or an
+  // opaque unique identifier.
+  uint32_t processor_id;
```

Comment:
> Since we aren't stable yet we're free to change the fields for now - what's here can be considered a template.

FYI this buildkite pipeline tests the latest release and latest source code together and it spotted a break earlier from the original PR: https://buildkite.com/iree/iree-samples/builds/460#0182f142-c462-4fa8-8963-b653bf4ef7a6
```
[2022-08-31T00:36:06Z] + /usr/src/github/iree/build-samples/samples/variables_and_state/variables-and-state /tmp/iree/colab_artifacts/counter_vmvx.vmfb local-task
[2022-08-31T00:36:06Z] ./samples/variables_and_state/test.sh: line 31: 14169 Segmentation fault      (core dumped) ${BUILD_DIR}/samples/variables_and_state/variables-and-state ${ARTIFACTS_DIR}/counter_vmvx.vmfb local-task
```
(that resolved itself after the next release brought the two back in sync)